src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_twin_valkyr.cpp

```diff
@@ -275,7 +275,9 @@ struct boss_twin_valkyrAI : public ScriptedAI
         if( me->HasUnitState(UNIT_STATE_CASTING) )
             return;
 
-        switch( events.ExecuteEvent() )
+        uint8 eventId = events.ExecuteEvent();
+
+        switch( eventId )
```

Comment:
https://www.azerothcore.org/wiki/cpp-code-standards#whitespace

Reply:
Actually whole script is written that way I've just changed events.ExecuteEvent() to evenId, normally I use switch (eventId)... should I change it and what is the fastest way?

Reply:
Its fine, I was just lazy and put the link.

What I mean is that you shouldnt have the whitespaces within the bracket